File: app/src/main/java/cc/smartcash/wallet/Services/CurrentPricesService.java
package cc.smartcash.wallet.Services;

import com.fasterxml.jackson.databind.JsonNode;

import retrofit2.Call;
import retrofit2.http.GET;

public interface CurrentPricesService {

    @GET("wallet/getcurrentpricewithcoin")
    Call<JsonNode> getCurrentPrices();
}

File: app/src/main/java/cc/smartcash/wallet/Services/CurrentPricesService.kt
package cc.smartcash.wallet.Services

import com.fasterxml.jackson.databind.JsonNode

import retrofit2.Call
import retrofit2.http.GET

interface CurrentPricesService {

    @get:GET("wallet/getcurrentpricewithcoin")
    val currentPrices: Call<JsonNode>
}